### DISCUSSION TITLE
Easy way to get current migration state

### DISCUSSION THREAD

**Discussion by tiloio** [pocketbase/pocketbase#3501] [Q&A] [answered]
Is there an easy way to check which migration is currently "active"?

I only found the following, which implies that we have to have sqlite3 installed...

@pgswow If you are still not able to apply the migrations on the prod server, you can:

- make sure that both environments has the same `pb_migrations` files content.
- run `sqlite3 pb_data/data.db 'select * from _migrations'` on both servers to check which migration files were applied

_Originally posted by @ganigeorgiev in https://github.com/pocketbase/pocketbase/discussions/1247#discussioncomment-4398522_

**Comment by ganigeorgiev** [comment, marked as answer]
> which migration is currently "active"?

I'm not sure that I understand what do you mean by this. All system and user defined migrations are automatically applied on serve. They are all "active" in the sense that they are applied.

There is no builtin command that will print which migration files are not applied, if that is what you are looking for, but if you don't want to use the sqlite3 cli tool you can create a View collection from the Admin UI with the following query that will print all applied migration files:

```sql
SELECT (ROW_NUMBER() OVER()) AS id,
       file,
       applied
FROM _migrations
ORDER BY applied ASC
```
